src/ray/raylet/scheduling_queue.cc

```diff
@@ -45,28 +45,72 @@ namespace ray {
 
 namespace raylet {
 
+SchedulingQueue::TaskQueue::~TaskQueue() {
+  task_map_.clear();
+  task_list_.clear();
+}
+
+bool SchedulingQueue::TaskQueue::AppendTask(const TaskID &task_id, const Task &task) {
+  RAY_CHECK(task_map_.find(task_id) == task_map_.end());
+  auto list_iter = task_list_.insert(task_list_.end(), task);
+  task_map_[task_id] = list_iter;
+  return true;
+}
+
+bool SchedulingQueue::TaskQueue::RemoveTask(const TaskID &task_id) {
+  auto iter = task_map_.find(task_id);
```

Comment:
instead of `iter` how about `task_found_iterator`

Reply:
revised.